### PR TITLE
[VAN-125800] integration tests

### PR DESCRIPTION
**Pull request by Shmalikov**
Added several integration tests. 
Other tests would be added while finding errors.

### CODE REVIEW

File: tests/integration/test_data_version_export_api.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+from visier_api_core import Configuration, ApiClient
+from visier_api_data_out.api.data_version_export_api import DataVersionExportApi
+
+
+class TestDataVersionExportApi(unittest.TestCase):
+    """DataVersionExportApi unit test stubs"""
+
+    def setUp(self) -> None:
+        load_dotenv()
```
[the remark this answers is not in the document]
Comment (Shmalikov) [reply tests/integration/test_data_version_export_api.py:26]: Forget to remove old ones.

File: tests/test-requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+visier-api-data-in==0.99201.1476
```
[the remark this answers is not in the document]
Comment (Shmalikov) [reply tests/test-requirements.txt:1]: updated test-requirements

File: integration_tests/visier_api_administration/test_permissions_api.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
+# coding: utf-8
+
+"""
+    Visier Administration APIs
+
+    Visier APIs for managing your tenant or tenants in Visier. You can programmatically manage user accounts in Visier, the profiles and permissions assigned to users, and to make changes in projects and publish projects to production. Administrating tenant users can use administration APIs to manage their analytic tenants and consolidated analytics tenants.<br>**Note:** If you submit API requests for changes that cause a project to publish to production (such as assigning permissions to users or updating permissions), each request is individually published to production, resulting in hundreds or thousands of production versions. We recommend that you use the `ProjectID` request header to make changes in a project, if `ProjectID` is available for the API endpoint.
+
+    The version of the OpenAPI document: 22222222.99201.1474
```
[the remark this answers is not in the document]
Comment (Shmalikov) [reply integration_tests/visier_api_administration/test_permissions_api.py:8]: Removed headers from files.

File: integration_tests/visier_api_administration/test_permissions_api.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    Contact: alpine@visier.com
```
[the remark this answers is not in the document]
Comment (Shmalikov) [reply integration_tests/visier_api_administration/test_permissions_api.py:9]: Removed headers from test files.

File: integration_tests/__init__.py
[the remark this answers is not in the document]
Comment (Shmalikov) [reply integration_tests/__init__.py:1]: Deleted.

File: integration_tests/utils.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import os
+
+from dotenv import load_dotenv
+
+from visier_api_core import Configuration, ApiClient
+
+TENANT = 'd3m'
+TENANT_CODE = 'WFF_d3m'
```
[the remark this answers is not in the document]
Comment (Shmalikov) [reply integration_tests/utils.py]: Fixed to getting from os env vars.

File: integration_tests/utils.py [hunk 6]
```diff
@@ -0,0 +1,33 @@
+import os
+
+from dotenv import load_dotenv
+
+from visier_api_core import Configuration, ApiClient
+
+TENANT = 'd3m'
```
[the remark this answers is not in the document]
Comment (Shmalikov) [reply integration_tests/utils.py]: deleted

File: tests/test-requirements.txt [hunk 7]
```diff
@@ -0,0 +1,7 @@
+python-dotenv>=1.0.1
```
Comment (hareskoug) [inline tests/test-requirements.txt:1]: We don't need to refer to the sdk packages here?

**Review by hareskoug — APPROVED** [review]

File: tests/test-requirements.txt (hunk 7 above)
Comment (Shmalikov) [reply tests/test-requirements.txt:1]: No, we need only add package folders to the PYTHONPATH or set them in IDE as sources root.

**Pull request merged**